ibis/expr/rules.py

```diff
@@ -709,6 +710,10 @@ def time(**arg_kwds):
     return ValueTyped(ir.TimeValue, 'not time', **arg_kwds)
 
 
+def interval(**arg_kwds):
+    return ValueTyped(ir.IntervalValue, 'not an interval', **arg_kwds)
+
+
 def timedelta(**arg_kwds):
```

Comment:
I wonder if this can be removed (or just aliased to `interval` for backwards compat)